### ISSUE TITLE
fontspec

### ISSUE THREAD

**Issue by FilipAndersson245** [ccebinger/CTAN_Inter#2]
How should I do to replace an existing font using fontspec with this?
I am trying to change the font of an existing LaTeX codebase and they do something like this to set the font.
I initially just added the `.otf` files and changed Roboto to Inter but i was unable to make it compile then, hoping this package can solve my problem.
```
\newfontfamily\headerfont[
  Path=\@fontdir,
  UprightFont=*-Regular,
  ItalicFont=*-Italic,
  BoldFont=*-Bold,
  BoldItalicFont=*-BoldItalic,
]{Roboto}
```

**Comment by ccebinger** [comment]
if you use awesome-cv, maybe this helps you:
https://tex.stackexchange.com/questions/345803/how-to-change-the-font-in-awesome-cv

Don't hesitate to contact me via email. My email-address is noted in the ReadMe.

**Comment by FilipAndersson245** [comment]
Yes Awsome-CV is what I am using, I did initially look into modifying the cls file to add some of the changes.
But I did not manage to make it work, https://pastebin.com/btZGx7LL <- is the output i got then trying to compile just changing `Roboto -> Inter`, with the files otf files added.
